rulebook.tex

```diff
@@ -407,18 +413,19 @@ \subsection{The Task}
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \subsection{Agreements \& Regulations}
 \label{sec:agreements}
-\paragraph{Standardized Robotic Platform}
-In the \ac{RCLL}, the Robotino robotic system from Festo Didactic SE is used as
-the common base platform.
-It can be used with certain freedoms and limitations regarding hardware and
-software modifications and additions, which will be outlined in
-\refsec{sec:robotino}.
-This includes the current version, Robotino 4, as well as the phased-out
-Robotino 3 and 2.
-Entry-level teams can explore the league with other robotic platforms as
-participation in the challenge track does not require the usage of the
-Robotino platform. However, the general hardware constraints outlined in
-\refsec{sec:robotino} still apply.
+\paragraph{Robotic Platforms}
+Until 2025, the Festo Robotino was the standard platform
+of the \ac{RCLL}. Starting in 2025, the \ac{RCLL} will allow the usage of
+alternative robotic platforms for every track of the competition, as long as
+they fulfill the general hardware constraints outlined in \refsec{sec:robotino}.
+The \ac{RCLL} will continue to support the Robotino platform as a reference
+platform for the main track of the competition, but teams are encouraged to
+explore the league with other robotic platforms as well.
+
+Hardware constraints for the main track will be derived from the specifications
```

Comment:
shouldnt we make this the same for challenge and main track? Would make it simpler and I see no real benefit for treading the tracks separately